tensorboard/tools/license_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+files=$(grep -rL "Copyright 20[0-9][0-9] The TensorFlow" \
```

Comment:
Hmm, it's not really good practice for a bazel-built script to have undeclared dependencies on local files like this.

Since the intent is just to run this script on travis, there's not much reason to run this as an actual bazel test.  Can we instead just make it a dedicated "before script" action configured in our .travis.yml file, like the flake8 tests, by running the script itself, not via bazel test?  That way it's clear that it's meant as a script external to bazel, rather than a nonconforming bazel test.